common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1627,14 +1627,15 @@ def test_timed_multi_attempt(self):
         module = CapaFactory.create(minutes_allowed=5, attempts=3)
         module.start_problem()
 
-        with patch('capa.correctmap.CorrectMap.is_correct') as mock_is_correct, \
-             patch('capa.capa_problem.LoncapaProblem.grade_answers') as mock_grade:
-
+        with (
+            patch('capa.correctmap.CorrectMap.is_correct') as mock_is_correct,
```

Comment:
I think your Jenkins test build is failing due to a syntax error in this block.

Do your tests pass when run locally?